accepted/0000-npm7-support-package-lock-6.md

```diff
@@ -0,0 +1,59 @@
+# Support `package-lock.json` v3 in npm 7
+
+## Summary
+
+Currently npm 7 can read `package-lock.json` versions 1, 2, and 3, but it always writes version 3. It would be nice if npm 7 could write `package-lock.json` version 3 instead.
```

Comment:
```suggestion
Currently npm 7 can read `package-lock.json` versions 1, 2, and 3, but it always writes version 2. It would be nice if npm 7 could write `package-lock.json` version 3 instead.
```
?